client/js/router/index.js

```diff
@@ -5,22 +5,25 @@ import LoginSignup from '../views/LoginSignup.vue';
 import Lobby from '../views/Lobby.vue';
 import GameView from '../views/GameView.vue';
 import Rules from '../views/Rules.vue';
+import Stats from '../views/Stats.vue';
 import store from '../store/store.js';
 
 Vue.use(VueRouter);
 
+const mustBeAuthenticated = (to, from, next) => {
+  if (store.state.auth.authenticated) {
+    next();
+  } else {
+    next('/login');
+  }
```

Comment:
I think you can do this to avoid the if/else:
```suggestion
  if (store.state.auth.authenticated) {
    return next();
  }
  return next('/login');
```

Reply:
Yeah agreed. Prefer the return w/o else